bin/plot_chpi_snr.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Plot signal-to-noise of continuous HPI coils as a function of time.
+Works by fitting a general linear model (HPI freqs, line freqs, DC, slope) to
+the data, and comparing estimated HPI powers with the residual (=variance
+unexplained by the model).
+Window length for SNR estimates can be specified on the command line.
+Longer windows will by nature include more low frequencies and thus have
+larger residual variance (lower SNR).
+"""
+
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir))
```

Comment:
The idea was to be able to run the script either from bin/ or the parent dir. If `setup.py` takes care of this, I guess the path append is not needed.